automated-tests/src/test/resources/cucumber/network/ethernetNetwork.feature

```diff
@@ -70,6 +70,17 @@ Feature: In order to manage Ethernet Networks
       And OneView gets the Associated Uplink Groups
     Then I get a count
 
+  @patch
+  Scenario: Update an Ethernet Network by Patch
+    Given name "network-bdd" for Resource
+      And Resource values will be updated as follows:
+      | op    | replace      |
+      | path  | /scopeUris   |
+      | value | Sample Scope |
+    When OneView gets Resource by Name
+      And OneView runs Ethernet Network patch
+    Then Resource is found
```

Comment:
I think we should change the verification for a success status, or to verify the resources properties (scopeUris attribute).

Reply:
Done.